.github/workflows/docs.yml

```diff
@@ -50,6 +50,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           ref: ${{ inputs.branch == '' && github.ref_name || inputs.branch }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
```suggestion
          # see https://github.com/actions/checkout/issues/551#issuecomment-1113089390
          repository: ${{ github.event.pull_request.head.repo.full_name }}
```

Reply:
Shouldn't this be synced up with copier template too then?

Reply:
This was a suggestion, you can of course take it or leave it.